.travis.yml

```diff
@@ -131,6 +132,12 @@ after_success:
       codecov; \
     fi
   # can't use test here since this leads to travis fails even though the build passes
-  - if [[ ${TRAVIS_PULL_REQUEST} == "false" ]] && [[ ${BUILD_DOCS} == "true" ]] && [[ ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} ]]; then
-            bash ${TRAVIS_BUILD_DIR}/maintainer/deploy_docs.sh;
+  - |
+    if [[ ${TRAVIS_PULL_REQUEST} == "false" ]] && [[ ${BUILD_DOCS} == "true" ]] && [[ ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} ]]; then
+      python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
```

Comment:
Just out of curiosity: why is this needed?

Reply:
It was due to this error, and the solution is from there: https://github.com/travis-ci/travis-ci/issues/4704. The build kept erroring from an unavailable resource; I think what was happening was that I had multiple branches going at once (1.0.0 and 2.0.0-dev0), causing the other branch to error instead of waiting its turn, but I didn't precisely narrow down the cause per se.

Reply:
Thanks for the explanation. Perhaps add a comment with a link to https://github.com/travis-ci/travis-ci/issues/4704 ?

Kudos to you for finding the fix!